control-plane/cmd/webhook-kafka/main.go

```diff
@@ -140,7 +140,7 @@ func NewValidationAdmissionController(ctx context.Context, cmw configmap.Watcher
 		ctxFunc,
 
 		// Whether to disallow unknown fields.
-		true,
+		false,
```

Comment:
The problem occurs after downgrade to 1.33. I'm wondering if branch 1.14 is the right place. Should this be 1.13? (but that would require a patch release for older version).

Reply:
You're right

Reply:
but then we need a patch release?

Reply:
https://github.com/openshift-knative/eventing-kafka-broker/pull/1297

Reply:
at least that will fix the midstream upgrade test